Fabric.Identity.API/scripts/Install-Identity-Utilities.psm1

```diff
@@ -822,7 +822,7 @@ function Get-SettingsFromInstallConfig {
         [string] $setting
     )
 
-    return Get-TenantSettingsFromInstallConfig($installConfigPath, $scope, $setting)
+    return Get-TenantSettingsFromInstallConfig -installConfigPath $installConfigPath -scope $scope -setting $setting
```

Comment:
Did this fix a bug, or just easier to read?

Reply:
![image](https://user-images.githubusercontent.com/30940601/55987175-26413680-5c5e-11e9-8d3d-611dfc697d6e.png)

Fixed a bug.  Was getting this error when running the script prior to the refactor